style/sass/README.md

```diff
@@ -45,10 +45,9 @@
 * Use ID and class names that are as short as possible but as long as necessary.
 * Avoid using the direct descendant selector `>`.
 * Avoid nesting more than 3 selectors deep.
-* Use HTML tags on vague classes that need a qualifier like `header.application` not `.main`.
-* Avoid using the HTML tag in the class name: `section.news` not `section.news-section`.
-* Avoid using HTML tags on classes for generic markup `<div>`, `<span>`: `.widgets` not `div.widgets`.
-* Avoid using HTML tags on classes with specific class names like `.featured-articles`.
+* Avoid over-qualified selectors:
```

Comment:
I sorta figured that. I would say either be specific about what we consider over qualifying or remain vague in both the statement and example. What about wrapping this and the direct descendant up together (both are over-qualifying)? 

Remove line 46 too and add
```
* Avoid over-qualified and over-specific selectors (`h1.page-title`, `div > .page-title`)
```